src/plugins/irc-events/invite.js

```diff
@@ -0,0 +1,29 @@
+var _ = require("lodash");
+var Msg = require("../../models/msg");
+
+module.exports = function(irc, network) {
+	var client = this;
+	irc.on("invite", function(data) {
+		var target = data.to;
+		if (target.toLowerCase() === irc.me.toLowerCase()) {
```

Comment:
While I can't find any RFC that defines nicknames as case-insensitive, [RFC 1459](https://tools.ietf.org/html/rfc1459#section-2.2) does point out the special casing imposed by IRC's Scandinavian origins (the `{}|` and `[]\` sets of characters).